src/main/java/nl/novi/TechItEasy/Controllers/TelevisionsController.java

```diff
@@ -0,0 +1,61 @@
+package nl.novi.TechItEasy.Controllers;
+
+import nl.novi.TechItEasy.Exceptions.RecordNotFoundException;
+import nl.novi.TechItEasy.Exceptions.RecordTooLongException;
+import nl.novi.TechItEasy.Model.Tv;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("tvs")
+public class TelevisionsController {
+private List<Tv> tvsDataBase = new ArrayList<>();
+
+    @GetMapping("")
+    public ResponseEntity<Object> getTvs () {
+        return new ResponseEntity<>(tvsDataBase, HttpStatus.OK);
+    }
+
+@GetMapping("/{id}")
+public ResponseEntity<Object> getTvById(@PathVariable int id) {
+//        if (id < 0 || id >= tvs.size()){
+//            return new ResponseEntity<>("Not exist", HttpStatus.NOT_FOUND);
+//        }
+    if (id < 0 || id >= tvsDataBase.size()) { // determine if id exists
+        throw new RecordNotFoundException("ID cannot be found");
+    }
+        return ResponseEntity.ok().body(tvsDataBase.get(id));
+}
+
+    @PostMapping("")
+    public ResponseEntity<Tv> addTv (@RequestBody Tv ntv) {
```

Comment:
Misschien is het verstandig om betekenisvollere variabele namen te kiezen. Over een half jaar kom je terug bij deze code en denk je "wat is ntv?". Nu is je code nog klein, dus kun je het zo terug vinden, maar als je code groter is, dan wordt dat een probleem.